html/webrtc.html

```diff
@@ -88,29 +109,16 @@
           return response.json();
         }).then(function(answer) {
           pc.remote_pc_id = answer.id;
+          console.log("Setting remote SDP offer")
           return pc.setRemoteDescription(answer);
         }).then(function() {
           return pc.createAnswer();
         }).then(function(answer) {
+          console.log("Setting local SDP answer")
           return pc.setLocalDescription(answer);
-        }).then(function() {
-            // wait for ICE gathering to complete
-            return new Promise(function(resolve) {
-                if (pc.iceGatheringState === 'complete') {
-                    resolve();
-                } else {
-                    function checkState() {
-                        if (pc.iceGatheringState === 'complete') {
-                            pc.removeEventListener('icegatheringstatechange', checkState);
-                            resolve();
-                        }
-                    }
-                    pc.addEventListener('icegatheringstatechange', checkState);
-                }
-            });
         }).then(function(answer) {
+          console.log("Sending SDP answer")
           var offer = pc.localDescription;
-
           return fetch(window.location.href, {
```

Comment:
Wouldn't it be easier to gather ICE candidates as part of `new Promise(function(resolve) {` and send them in this request instead of introducing new one?

We parse `pending_remote_candidates` in there.